### PR TITLE
docs: add headers

### LINKED ISSUE #5848: Add headers to Git articles, part 1
### What article on docs.github.com is affected?

- "[Setting your username in Git](https://docs.github.com/en/github/getting-started-with-github/setting-your-username-in-git)"
   - New header: About Git usernames
- "[Configuring git to handle line endings](https://docs.github.com/en/github/getting-started-with-github/setting-your-username-in-git)
  - New header: About line endings
- "[Pushing commits to a remote repository](https://docs.github.com/en/github/getting-started-with-github/pushing-commits-to-a-remote-repository)"
  - New header: About `git push`

### What part(s) of the article would you like to see updated?

There's not currently a header above the initial information in the articles, so the information isn't represented or linkable in the article's table of contents. Add H3 headers to the beginning of the articles (see suggested header titles above).

### Additional information

This issue is for a special project and is not currently available for contributors to work on.

### PR DESCRIPTION
**Pull request by khattakdev**
### Why:

Closes https://github.com/github/docs/issues/5848

### What's being changed:

image
image
image

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by RupeshBhandari** [comment]
the headers would be super useful

**Comment by ramyaparimi** [comment]
@khattakdev Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/get-started/using-git/pushing-commits-to-a-remote-repository.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ versions:
   enterprise-server: '*'
   github-ae: '*'
 ---
+## `git push`
```
Comment (mchammer01) [inline content/get-started/using-git/pushing-commits-to-a-remote-repository.md:15]:
```suggestion
## About `git push`
```

**Review by mchammer01 — DISMISSED** [review]
@khattakdev - this LGTM ✨ 
I've added a suggestion that I'm going to apply.

**Review by mchammer01 — APPROVED** [review]
Re-approving

**Pull request merged**